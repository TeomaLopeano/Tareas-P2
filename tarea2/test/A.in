cantidadEnCadena
estaEnCadena 5
insertarAlInicio 5 5.0
primeroEnCadena
imprimirCadena
estaEnCadena 5
infoCadena 5
removerDeCadena 5
cantidadEnCadena
insertarAlInicio 9 9.0
insertarAlInicio 8 8.0
insertarAlInicio 7 7.0
insertarAlInicio 6 6.0
imprimirCadena
primeroEnCadena
infoCadena 9
infoCadena 7
infoCadena 6
removerDeCadena 7
removerDeCadena 6
removerDeCadena 9
primeroEnCadena
Fin