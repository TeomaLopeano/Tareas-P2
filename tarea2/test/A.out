1>La cantidad es 0.
2>5 no está en la cadena.
3>Se insertó (5,5.00) al inicio de la cadena.
4>El primero es (5,5.00)
5>(5,5.00)
6>5 está en la cadena.
7>El elemento es (5,5.00)
8>Se removió la primera aparición de 5 de la cadena.
9>La cantidad es 0.
10>Se insertó (9,9.00) al inicio de la cadena.
11>Se insertó (8,8.00) al inicio de la cadena.
12>Se insertó (7,7.00) al inicio de la cadena.
13>Se insertó (6,6.00) al inicio de la cadena.
14>(6,6.00)(7,7.00)(8,8.00)(9,9.00)
15>El primero es (6,6.00)
16>El elemento es (9,9.00)
17>El elemento es (7,7.00)
18>El elemento es (6,6.00)
19>Se removió la primera aparición de 7 de la cadena.
20>Se removió la primera aparición de 6 de la cadena.
21>Se removió la primera aparición de 9 de la cadena.
22>El primero es (8,8.00)
23>Fin.
